src/quickstart/provider/azure-acs/dns.md

```diff
@@ -0,0 +1,49 @@
+## Find Your Load Balancer Address
+
+On Azure Container Engine, Deis Workflow will automatically provision and
+attach a Azure Load Balancer to the router component. This component is
+responsible for routing HTTP and HTTPS requests from the public internet to
+applications that are deployed and managed by Deis Workflow.
+
+By describing the `deis-router` service, you can see what IP address has been
+allocated by Azure Cloud for your Deis Workflow cluster:
+
+```
+$ kubectl --namespace=deis get service deis-router
+NAME          CLUSTER-IP    EXTERNAL-IP    PORT(S)                            AGE
+deis-router   10.0.60.172   13.82.148.57   80/TCP,443/TCP,2222/TCP,9090/TCP   54m
+```
+
+TODO: mention `<pending>` state
+
+## Prepare the Hostname
+
+Now that you have the ip address of your load balancer we can use the `nip.io`
```

Comment:
The goal here is a quickstart... and using nip/xip don't ring to me as a recommendation for production use.

We can have a DNS recommendation section, but that is out of scope for this PR.